website/source/docs/provisioning/shell.html.md

```diff
@@ -130,9 +130,9 @@ end
 ```
 
 In the code block above, the script block starts with `<<-SCRIPT` and ends with `SCRIPT`.
-This is known as a _Here Document_ or a _heredoc_.  Additionally, if your script
+This is known as a `Here Document` or a `heredoc`.  Additionally, if your script
```

Comment:
This is known as a "here document" or a "heredoc".  Additionally, if your script